sites/feedback/includes/feedback.html

```diff
@@ -2,14 +2,24 @@
 	{%- if page.feedbackPath -%}
 		{{ page.feedbackPath }}
 	{%- elsif site.global.feedbackPath and site.global.feedbackPath.first -%}
-		{{ site.global.feedbackPath[ i18nText-lang ] | relative_url }}
+		{{ site.url }}{{ site.baseurl }}{{ site.global.feedbackPath[ i18nText-lang ] }}
 	{%- endif -%}
 {%- endcapture -%}
 
 
 <div {% if feedbackPath and feedbackPath != "" -%}
-		data-ajax-replace="{{feedbackPath}}"
-	{%- endif -%}>
+		data-ajax-replace="{{feedbackPath}}" class="wb-disable-allow"
```

Comment:
Make the condition "wb-disable-allow" optional for our RAP example

Idem for the anchor, remove it by default but keep it for our RAP example